### PR TITLE
General deployment faq/del opt

### PR DESCRIPTION
**Pull request by BobleChinois**
A simple proposal for the FAQ based on @JeffVandrewJr answer to issue #214

### CODE REVIEW

File: FAQ/FAQ-Deployment.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Here are common questions about installation, regardless of the deployment metho
 * [With the docker deployment, how to use a different volume for the data?](FAQ-Deployment.md#with-the-docker-deployment-how-to-use-a-different-volume-for-the-data)
 * [How do I activate Tor on my BTCPay Server?](FAQ-Deployment.md#how-do-i-activate-tor-on-my-btcpay-server)
 * [Why activate Tor? Does it mean that nobody knows who I am?](FAQ-Deployment.md#why-activate-tor-does-it-mean-that-nobody-knows-who-i-am)
+* [How can I deactivate Tor (or any other option)?](FAQ-Deployment.md#how-can-i-deactivate-tor-or-any-other-option)
```
Comment (pavlenex) [inline FAQ/FAQ-Deployment.md]: `or any other option` is unclear, nobody will search for how to deactivate any other option. I think you should re-write this to be plugin neutral and provide Tor as an example.

Comment (pavlenex) [reply FAQ/FAQ-Deployment.md]: I think:
How to deactivate environment variable in BTCPay?

Comment (BobleChinois) [reply FAQ/FAQ-Deployment.md]: Mmmhhh not sure if "environment variable" will be very clear for the average user. Do we have a complete list of all the possible options somewhere?

Comment (BobleChinois) [reply FAQ/FAQ-Deployment.md]: otoh people that messing with the command line like this probably know about it

Comment (britttttk) [reply FAQ/FAQ-Deployment.md]: yes here is the complete list: https://github.com/btcpayserver/btcpayserver-docker#environment-variables

Comment (BobleChinois) [reply FAQ/FAQ-Deployment.md]: Ok cool, seems like we could add `opt-add-tor` to this list. Thanks! I'll get back to this later

Comment (britttttk) [reply FAQ/FAQ-Deployment.md]: No, opt-add-tor is below under the BTCPAYGEN_ADDITIONAL_FRAGMENTS list. Might be a good idea to link the "Warning" to your "Why activate Tor?" though :)

**Pull request merged**